x-pack/plugins/aiops/public/components/change_point_detection/chart_component.tsx

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { type TypedLensByValueInput } from '@kbn/lens-plugin/public';
+import { useChangePointDetectionContext } from './change_point_detection_context';
+import { useDataSource } from '../../hooks/use_data_source';
+import { useAiopsAppContext } from '../../hooks/use_aiops_app_context';
+import { useTimeRangeUpdates } from '../../hooks/use_time_filter';
+import { fnOperationTypeMapping } from './constants';
+
+export interface ChartComponentProps {
+  annotation: {
+    group_field: string;
+    label: string;
+    timestamp: string;
+    reason: string;
+  };
+}
+
+export const ChartComponent: FC<ChartComponentProps> = React.memo(({ annotation }) => {
```

Comment:
@flash1293 Could you please review this file and verify if usage of the Lens embeddable is correct? Especially the attributes part